tests/csapi/room_relations_test.go

```diff
@@ -12,11 +12,9 @@ import (
 	"github.com/matrix-org/complement/internal/client"
 	"github.com/matrix-org/complement/internal/match"
 	"github.com/matrix-org/complement/internal/must"
-	"github.com/matrix-org/complement/runtime"
 )
 
 func TestRelations(t *testing.T) {
-	runtime.SkipIf(t, runtime.Dendrite) // not supported
```

Comment:
This seems to be #499?

Reply:
I did not spot that, should I remove these changes so they don't conflict?

Reply:
I guess leave it, they'll conflict and they can fix it there.